### PR TITLE
feat(reference/architecture): First pass at overview and 'Life of a D…

### PR DESCRIPTION
**Pull request by duftler**
…eployment' sequence diagram.

### CODE REVIEW

**Comment by duftler** [comment]
@lwander @ttomsu @skim1420 please review.

File: reference/architecture/index.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
+layout: single
+title:  "Architecture"
+sidebar:
+  nav: reference
+---
+
+{% include toc %}
+
+## Spinnaker Microservices
```
Comment (skim1420) [inline reference/architecture/index.md:10]: Add fiat?

Comment (duftler) [reply reference/architecture/index.md:10]: Done.

**Review by lwander — APPROVED** [review]
LGTM - igor depends on clouddriver for images

**Comment by duftler** [comment]
Added the igor->clouddriver dependency.

**Pull request merged**